applications/frontend/src/store/reducers/nwbfiles.ts

```diff
@@ -0,0 +1,32 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit'
+
+import { CallApiAction } from '../../middleware/backend'
+import { NWBFile } from '../../types/nwbfile'
+
+const initialState: NWBFile[] = [];
+
+const NWBFileSlice = createSlice({
```

Comment:
Why there is a whole logic around nwb files? Probably we can generalize this to workspace resources in general

Reply:
because it's in the mockup

Reply:
explained